Source/Core/Core/HW/MMIO.h

```diff
@@ -62,9 +62,10 @@ inline bool IsMMIOAddress(u32 address)
 // The block ID can easily be computed by simply checking bit 24 (CC vs. CD).
 inline u32 UniqueID(u32 address)
 {
-  _dbg_assert_msg_(MEMMAP, ((address & 0xFFFF0000) == 0x0C000000) ||
-                               ((address & 0xFFFF0000) == 0x0D000000) ||
-                               ((address & 0xFFFF0000) == 0x0D800000),
+  _dbg_assert_msg_(MEMMAP,
+                   ((address & 0xFFFF0000) == 0x0C000000) ||
+                       ((address & 0xFFFF0000) == 0x0D000000) ||
+                       ((address & 0xFFFF0000) == 0x0D800000),
```

Comment:
I don't quite understand why those cannot line up in any perceivable way...are we missing any options in our clang-format files? Or is it that both versions don't know what to do with this (neither the old nor the new one look "aligned" in any way to me).